README.md

```diff
@@ -443,33 +443,34 @@ of the time a higher-level object will suffice as an argument.
 **Bad:**
 
 ```php
-function createMenu(string $title, string $body, string $buttonText, bool $cancellable): void
+function createMenu(string $title = '', string $body = '', string $buttonText = '', bool $cancellable = false): void
 {
     // ...
 }
+
+createMenu('Foo', '', '', true);
 ```
 
 **Good:**
 
 ```php
 class MenuConfig
 {
-    public $title;
-    public $body;
-    public $buttonText;
+    public $title = '';
+    public $body = '';
+    public $buttonText = '';
     public $cancellable = false;
 }
 
-$config = new MenuConfig();
-$config->title = 'Foo';
-$config->body = 'Bar';
-$config->buttonText = 'Baz';
-$config->cancellable = true;
-
 function createMenu(MenuConfig $config): void
 {
     // ...
 }
+
+$config = new MenuConfig();
```

Comment:
We should ensure that MenuCOnfig is in a valid state too.
So sth like

```

class DTO
{
    /**
     * @var string
     */
    private $title;

    /**
     * @var string
     */
    private $body;

    /**
     * @var string
     */
    private $buttonText;

    /**
     * @var bool
     */
    private $cancellable;

    public function __construct(string $title, string $body, string $buttonText, string $cancellable)
    {
        $this->title = $title;
        $this->body = $body;
        $this->buttonText = $buttonText;
        $this->cancellable = $cancellable;
    }
}
```

Reply:
Not. It is not required to be valid. Do not confuse DTO and VO.